File: JS/classes/uber-classes/specialUberCar.js
const UberCar = require("./uberCar").UberCar;

class SpecialUberCar extends UberCar{
    constructor(license,driver,typeCarAccepted,seatsMaterial){
        super(license,driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

exports.SpecialUberCar = SpecialUberCar;

File: Java/SpecialUberCar.java
import java.util.ArrayList;
import java.util.Map;

class SpecialUberCar extends UberCar{

    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public SpecialUberCar(String license, Account driver,Map<String, Map<String,Integer>> typeCarAccepted,ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}